app/code/Magento/Catalog/Controller/Adminhtml/Category/Add.php

```diff
@@ -6,31 +6,38 @@
  */
 namespace Magento\Catalog\Controller\Adminhtml\Category;
 
-use Magento\Framework\App\Action\HttpGetActionInterface as HttpGetActionInterface;
+use Magento\Backend\App\Action\Context;
+use Magento\Backend\Model\View\Result\Page;
+use Magento\Framework\Controller\ResultFactory;
+use Magento\Backend\Model\View\Result\Redirect;
+use Magento\Framework\Controller\ResultInterface;
+use Magento\Catalog\Controller\Adminhtml\Category;
+use Magento\Backend\Model\View\Result\ForwardFactory;
+use Magento\Framework\App\Action\HttpGetActionInterface;
```

Comment:
No need to revert, but please avoid importing FQCN mixed with other changes.

Reply:
> No need to revert, but please avoid importing FQCN mixed with other changes.

Why we should avoid importing FQCN mixed with other changes?